src/main/scala/org/constellation/consensus/Round.scala

```diff
@@ -67,8 +67,9 @@ class Round(
       throw new RuntimeException("Received message from different round.")
     case StartTransactionProposal(_) =>
       sendArbitraryDataProposalsTikTok.cancel()
+
       val transactions = dao.transactionService
-        .pullForConsensus(1)
+        .pullForConsensusIfEmptyCreateDummyTransaction(1)
```

Comment:
I think that the name is too verbose, what about `pullForConsensusWithDummy`?

Reply:
Or maybe leave `pullForConsensus` atomic, so just keep the functionality of pulling transactions and just check if there are transactions. If no, create a dummy transaction then